rasa/core/evaluation/marker_stats.py

```diff
@@ -1,5 +1,8 @@
 from __future__ import annotations
 from typing import Dict, Text, Union, List, Tuple
+
+from numpy import floating
```

Comment:
if it's just for typing purposes, I suggest using `if TYPE_CHECKING`. But in this case, `numpy` is imported below at line 10. So maybe using `np.floating` just works below?